packages/terra-search-field/src/SearchField.jsx

```diff
@@ -162,8 +167,19 @@ class SearchField extends React.Component {
   }
 
   updateSearchText(searchText) {
-    if (searchText !== undefined && searchText !== this.state.searchText) {
-      this.setState({ searchText });
+    if (searchText !== undefined && searchText !== this.searchText) {
```

Comment:
to cover all bases, is it worth updating this to a `typeof` check to protect from undeclared variables?
```suggestion
    if (typeof searchText !== 'undefined' && searchText !== this.searchText) {
```